ote/src/cljs/ote/views/transit_changes.cljs

```diff
@@ -190,19 +190,41 @@
                                       "Ei löydettyjä muutoksia")
                    :name->label str
                    :label-style (merge style-base/table-col-style-wrap {:font-weight "bold"})
-                   :stripedRows true
-                   :row-style {:cursor "pointer"}
-                   :show-row-hover? true
-                   :on-select (fn [evt]
-                                (let [{:keys [transport-service-id date]} (first evt)]
-                                  (e! (tc/->ShowChangesForService transport-service-id
-                                                                  date))))}
+                   :stripedRows true}
       [{:name "Palveluntuottaja"
-        :read :transport-operator-name
+        :read identity
+        :format (fn [row]
```

Comment:
Clicking via link instead of table row is used,
This `fn` should be defn'd and reused for Palveluntiuottaja and Palvelu